hoyo_buddy/cogs/profile.py

```diff
@@ -207,6 +207,7 @@ async def profile_zzz_command(
         account: app_commands.Transform[
             HoyoAccount | None, HoyoAccountTransformer(COMMANDS["profile zzz"].games)
         ] = None,
+        uid: app_commands.Range[str, 9, 10] | None = None,
```

Comment:
```suggestion
        uid: app_commands.Range[str, 8, 10] | None = None,
```

CN server's UID is 8 digit long